test/sddp.jl

```diff
@@ -117,16 +117,16 @@ using Test
         @test StochDynamicProgramming.ncuts(sddppr.bellmanfunctions) <= ncutini
     end
 
-    @testset "Quadratic regularization" begin
-        optimizer2 = optimizerQP
-        param2 = StochDynamicProgramming.SDDPparameters(optimizer2,
-                                                    passnumber=n_scenarios,
-                                                    gap=epsilon,
-                                                    max_iterations=max_iterations)
-        #TODO: fix solver, as Clp cannot solve QP
-        @test_throws ErrorException solve_SDDP(model, param2, 0, 0,
-                                                regularization=SDDPRegularization(1., .99))
-    end
+    # @testset "Quadratic regularization" begin
```

Comment:
Yes, I agree that it might be a good idea to remove this test. The quadratic regularization was never of great help anyway.

Reply:
Ok, let's keep them there then